mutable_tree.go

```diff
@@ -642,19 +639,29 @@ func (tree *MutableTree) enableFastStorageAndCommitIfNotEnabled() (bool, error)
 		return false, nil
 	}
 
-	if isFastStorageEnabled && shouldForceUpdate {
```

Comment:
I think we should leave this as some users may not want to upgrade at this time

Reply:
Once users update the iavl code version, it will auto uggrade to fast IAVL.

Reply:
there is another PR to make it optional since some dbs may take too long. If a node doesn't upgrade then would this still work?

Reply:
I glanced the pr#547. It seems if `skipFastStorageUpgrade` is true. it will not go here. so we do not need to readd the `isFastStorageEnabled && shouldForceUpdate`